### ISSUE TITLE
Ability to search bookmarks

### ISSUE THREAD

**Issue by nvx** [mapsme/omim#1429]
The ability to import kml bookmarks is great, but when importing a large number it becomes difficult to find bookmarks in the list as there is no search functionality that I could find.

**Comment by deathbaba** [comment]
Thanks, bookmarks search is already in our TODO list.

**Comment by mhyqwvcu** [comment]
Will this also add bookmarks to the route planning screen? Currently the only options are History and Categories. If you want to select a from or to point that is a bookmark you have to drag the shrunken screen around until you locate the point on a map or tap several times to open and locate it in the bookmark menu.

**Comment by igortomko** [comment]
This will be implemented in the future

**Comment by MagTun** [comment]
Question from the *future*: any news about this feature?

**Comment by burivuh** [comment]
Hi!

You have a local bookmark search inside any of your bookmarks lists.
Global search is not in the nearest plans.

**Comment by MagTun** [comment]
That's right but the feature I need is the "global search".  Hope in the future you will make it!
